echopype/utils/coding.py

```diff
@@ -53,6 +55,30 @@ def _encode_dataarray(da, dtype):
     return coding.times.decode_cf_datetime(encoded_data, **read_encoding)
 
 
+def _get_auto_chunk(
+    variable: xr.DataArray, chunk_size: "int | str | float" = "100MB"
+) -> Tuple[int]:
+    """
+    Calculate default chunks for a data array based on desired chunk size
+
+    Parameters
+    ----------
+    variable : xr.DataArray
+        The data array variable to be calculated
+    chunk_size : int or str or float
+        The desired max chunk size for the array.
+        Default is 100MB
+
+    Returns
+    -------
+    tuple
+        The chunks
+    """
+    auto_tuple = tuple(["auto" for i in variable.shape])
+    chunks = auto_chunks(auto_tuple, variable.shape, chunk_size, variable.dtype)
```

Comment:
@lsetiawan as we discussed today, I suggest specifying that chunks should occur along the time dimensions. This will be better for downstream methods.

Reply:
After conversation with @leewujung this is fine for the smaller files. The usecases for downstream combine echodata will be in a separate issue and PRs in the future.